META6.json

```diff
@@ -22,5 +22,5 @@
   "source-url" : "git://github.com/titsuki/p6-App-AizuOnlineJudge.git",
   "tags" : [ ],
   "test-depends" : [ ],
-  "version" : "0.0.2"
+  "version" : "0.0.3"
```

Comment:
I always use mi6 as a authoring tool and `mi6 release` command automatically increase the version field to the `0.0.3` from `0.0.2`. So, could you set this field to `0.0.2` as before?